CONTRIBUTUNG.md

```diff
@@ -0,0 +1,44 @@
+Please always report problems by filling an issue here on GitHub. This also covers reporting problems with the documentation (e.g. if sth is unclear).
+
+General rules:
+
+    1. the code style suggestions of PyCharm should be followed
+
+Job general:
+
+    1. If a class inherits directly/indirectly from the sisyphus class `Job` the class ends with "Job".
+    2. In a Sisyphus job class the output variables end in `out_*`
+    3. If a Job has non-trivial requirements it should have a `self.rqmt`
+
+Job constructor order:
+
+    1. set inputs
+    2. set outputs (should be prefixed with `out_`)
+    3. define rqmt
+
+Job function order:
+
+    1. `__init__`
+    2. `tasks()`
+    3. task functions
+    4. helper functions
+    5. class functions
+    6. `__hash__`
+
+Rasr-Jobs:
+should always accept `crp`, `extra_config` + `extra_post_config`
+
+Import order:
+
+    1. standard library
+    2. external libraries (including sisyphus, yes I know it's not like that atm)
```

Comment:
So Sisyphus not as first import anymore? I got quite used to it, but of course this is debatable.

Reply:
I thought it's easier to treat sisyphus as a normal third-party library. It also reduces the number of import blocks from 4 to 3.